controllers/pipeline/pipeline_adapter.go

```diff
@@ -113,6 +113,18 @@ func (a *Adapter) EnsureSnapshotExists() (reconciler.OperationResult, error) {
 		"snapshot.Name", expectedSnapshot.Name,
 		"snapshot.Spec.Components", expectedSnapshot.Spec.Components)
 
+	patch := client.MergeFrom(a.pipelineRun.DeepCopy())
+	h.AddAnnotation(&a.pipelineRun.ObjectMeta, tekton.SnapshotNameLabel, expectedSnapshot.Name)
+	err = a.client.Patch(a.context, a.component, patch)
+	if err != nil {
+		a.logger.Error(err, "Failed to update the build pipelineRun with new annotations",
+			"pipelineRun.Name", a.pipelineRun.Name)
+		return reconciler.RequeueWithError(err)
+	}
+	a.logger.LogAuditEvent("Updated build pipelineRun", a.pipelineRun, h.LogActionUpdate,
```

Comment:
Do we need this `"pipelineRun.Name", a.pipelineRun.Name,` in the a.logger.LogAuditEvent()? Others look good to me.

Reply:
You are right, that line is not needed because the resource name gets logged automatically. I removed it now, thanks.

Reply:
Thanks.